frontend/src/components/TooltipWrapper.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
```

Comment:
This was added so that the USWDS Tooltip could be loaded without SSR which is necessary to described here https://github.com/uswds/uswds/issues/4338 in which the tooltip loads with different IDs.